docs/rules/quote-props.md

```diff
@@ -1,5 +1,47 @@
 # Quoting Style for Property Names (quote-props)
 
+Object literal property names can be defined in two ways: using literals or using strings. For example, these two objects are equivalent:
+
+```js
+var object1 = {
+    property: true;
+};
+
+var object2 = {
+    "property": true
+};
+```
+
+In many cases, it doesn't matter if you choose to use an identifier instead of a string or vice-versa. There are, however, some occasions when you must use quotes:
+
+1. If you are using an ECMAScript 3 JavaScript engine (such as IE8) and you want to use a keyword (such as `if`) as a property name. This restriction was removed in ECMAScript 5.
+2. You want to use a non-identifier character in your property name, such as having a property with a space like `"one two"`.
+
+Sometimes the choice to use quotes is purely stylistic and sometimes it's functional. Here's another example:
+
+```js
+var object = {
+    1e2: 1,
+    100: 2
+};
+```
+
+This may look alright at first sight, but this code in fact throws a syntax error in ECMAScript 5 strict mode. This happens because `1e2` and `100` are coerced into strings before getting used as the property name. Both `String(1e2)` and `String(100)` happen to be equal to `"100"`, which causes the "Duplicate data property in object literal not allowed in strict mode" error. Issues like that can be tricky to debug, so some prefer to require quotes around all property names.
+
+## Rule Details
```

Comment:
Tests are failing because this occurs again later in the document. Looks like there might be some duplicated content.